assets/locales/en.json

```diff
@@ -1,825 +1,362 @@
 {
-	"about": {
```

Comment:
I think the whitespace got changed around when I removed the firmware_escrow bits from it